src/activities/Elsa.Activities.Http/Formatters/DefaultContentFormatter.cs

```diff
@@ -13,5 +13,10 @@ public Task<object> ParseAsync(byte[] content, string contentType)
         {
             return Task.FromResult<object>(content);
         }
+
+        public Task<byte[]> ParseAsync(object content, string contentType)
```

Comment:
It looks like your fork is out of date - the `IContentFormatter` no longer exists. It was replaced with `IHttpRequestBodyParser` and `IHttpResponseBodyParser`.

Reply:
Oh.... okay... ill check that